tree/ntuple/v7/src/RField.cxx

```diff
@@ -547,6 +548,8 @@ ROOT::Experimental::RFieldBase::Create(const std::string &fieldName, const std::
       return result;
    } else if (canonicalType == "std::string") {
       result = std::make_unique<RField<std::string>>(fieldName);
+   } else if (canonicalType == "TObject") {
+      result = std::make_unique<RTObjectField>(fieldName);
```

Comment:
should this be
```suggestion
      result = std::make_unique<RField<TObject>>(fieldName);
```
? (if not, I need to look into the distinction between `RTObjectField` and the `RField<>` specialization again)

Reply:
It shouldn't matter which one it is.

Reply:
Oh, now I understand: we don't need `RTObjectField` at all, just the template specialization. I'll fix that.